src/main/java/uk/gov/hmcts/probate/service/DocumentGeneratorService.java

```diff
@@ -41,56 +42,88 @@
     private static final String SEAL_FILE_PATH = "sealImage.txt";
     private static final String WATERMARK = "draftbackground";
     private static final String WATERMARK_FILE_PATH = "watermarkImage.txt";
+    private static final String DRAFT = "preview";
+    private static final String FINAL = "final";
 
-    private Map<String, Object> images = new HashMap<>();
+    private Map<String, Object> images;
```

Comment:
I dont think this should be an instance var on the class - just in the method needed - whats the reason for this on a service?